libs/core/shellbar/shellbar-branding/shellbar-branding.component.html

```diff
@@ -0,0 +1,20 @@
+<div
+    class="fd-shellbar__branding"
+    role="link"
+    tabindex="0"
+    [attr.aria-label]="ariaLabel"
+    (click)="callback ? callback($event) : ''"
+>
+    <span class="fd-shellbar__logo">
+        <ng-content select="fd-shellbar-logo"></ng-content>
+    </span>
```

Comment:
This results in:
```
<span class="fd-shellbar__logo">
    <fd-shellbar-logo>
        <a href="#" aria-label="SAP" class="fd-shellbar__logo fd-shellbar__logo--image-replaced"></a>
    </fd-shellbar-logo>
</span>
```

You have 2 elements with the same class `fd-shellbar__logo`. Not to mention that the parent element with class `fd-shellbar__branding` is clickable and inside you have a link (the logo), which can't pass the a11y requirements. You can't have a clickable element inside another clickable element.